docs/modules/ROOT/pages/apdx-operatorChoice.adoc

```diff
@@ -1,111 +1,111 @@
 [[which-operator]]
-= Which operator do I need?
+== Which operator do I need?
 
-TIP: In this section, if an operator is specific to https://projectreactor.io/docs/core/release/api/reactor/core/publisher/Flux.html[Flux] or https://projectreactor.io/docs/core/release/api/reactor/core/publisher/Mono.html[Mono], it is prefixed and linked accordingly, like this: https://projectreactor.io/docs/core/release/api/reactor/core/publisher/Flux.html#fromArray-T:A-[Flux#fromArray]. Common operators have no prefix, and links to both implementations are provided, for example: `just` (https://projectreactor.io/docs/core/release/api/reactor/core/publisher/Flux.html#just-T%2E%2E%2E-[Flux]|link:https://projectreactor.io/docs/core/release/api/reactor/core/publisher/Mono.html#just-T-[Mono]). When a specific use case is covered by a combination of operators, it is presented as a method call, with a leading dot and parameters in parentheses, as follows: `.methodCall(parameter)`.
+TIP: In this section, if an operator is specific to {javadoc}/reactor/core/publisher/Flux.html[Flux] or {javadoc}/reactor/core/publisher/Mono.html[Mono], it is prefixed and linked accordingly, like this: {javadoc}/reactor/core/publisher/Flux.html#fromArray-T:A-[Flux#fromArray]. Common operators have no prefix, and links to both implementations are provided, for example: `just` ({javadoc}/reactor/core/publisher/Flux.html#just-T%2E%2E%2E-[Flux]|link:{javadoc}/reactor/core/publisher/Mono.html#just-T-[Mono]). When a specific use case is covered by a combination of operators, it is presented as a method call, with a leading dot and parameters in parentheses, as follows: `.methodCall(parameter)`.
 
 //TODO flux:  publishOn/subscribeOn/cancelOn
 //transformDeferred/transform, repeatWhen, sort, startWith
 //TODO Mono.sequenceEqual
 
 I want to deal with:
 
-* <<which.create>>
+* xref:apdx-operatorChoice.adoc#which.create[Creating a New Sequence...]
 
-* <<which.values>>
+* xref:apdx-operatorChoice.adoc#which.values[Transforming an Existing Sequence]
 
-* <<which.filtering>>
+* xref:apdx-operatorChoice.adoc#which.filtering[Filtering a Sequence]
 
-* <<which.peeking>>
+* xref:apdx-operatorChoice.adoc#which.peeking[Peeking into a Sequence]
 
-* <<which.errors>>
+* xref:apdx-operatorChoice.adoc#which.errors[Handling Errors]
 
-* <<which.time>>
+* xref:apdx-operatorChoice.adoc#which.time[Working with Time]
 
-* <<which.window>>
+* xref:apdx-operatorChoice.adoc#which.window[Splitting a {javadoc}/reactor/core/publisher/Flux.html[Flux]]
 
-* <<which.blocking>>
+* xref:apdx-operatorChoice.adoc#which.blocking[Going Back to the Synchronous World]
 
-* <<which.multicasting>>
+* xref:apdx-operatorChoice.adoc#which.multicasting[Multicasting a {javadoc}/reactor/core/publisher/Flux.html[Flux] to several https://www.reactive-streams.org/reactive-streams-1.0.3-javadoc/org/reactivestreams/Subscriber.html?is-external=true[Subscribers]]
```

Comment:
Same comment as above for both links. It's ok if the actual header has these but the link to the header should be simple and be a single atomic link.

Reply:
Fixed in https://github.com/reactor/reactor-core/pull/3806/commits/bc98827e48210fa62a1bd0bdd9c44ab96c7d3d41